styles.css

```diff
@@ -304,6 +335,149 @@ h2 {
   color: #4472e7;
 }
 
+#contact-btn:hover {
+  color: white;
+  background-color: #4472e7;
+  border: 1px solid #4472e7;
+}
+
 .contact-message {
   height: 100px;
 }
+
+@media (min-width: 420px) {
+  .intro-section {
+    background-image: url('images/main-desktop-background.png');
+    height: 80vh;
+  }
+
+  .menu1 {
+    visibility: visible;
+    display: flex;
+  }
+
+  .menu1 ul {
+    display: flex;
+  }
+
+  .menu1 li {
+    list-style: none;
+    padding: 10px;
+    text-decoration: none;
+  }
+
+  .menu1 li a {
+    text-decoration: none;
+    color: #344563;
+    font-size: 1.2em;
+  }
+
+  .menu1 li a:hover {
+    color: #4472e7;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .contact-section {
+    background-image: url("images/contact-desktop-background.png");
+    background-size: cover;
+  }
+}
+
+@media (min-width: 750px) {
+  .languages-box2 {
+    padding-top: 40px;
+    width: 50%;
+  }
+
+  .language {
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    margin: 10px;
+    width: 8em;
+  }
+
+  #lang-list {
+    display: flex;
+  }
+
+  .language-text {
+    text-align: center;
+  }
+
+  .about {
+    width: 50%;
+    padding-top: 50px;
+  }
+
+  .about-section {
+    justify-content: space-around;
+    flex-direction: row;
+    padding: 50px;
+  }
+
+  header {
+    margin-left: 150px;
+    margin-right: 150px;
+  }
+
+  .menu {
+    visibility: hidden;
+  }
+
+  .contact-section {
+    width: 100vw;
+    background-image: url("images/contact-desktop-background.png");
+    background-size: cover;
+  }
+
+  .contact {
+    width: 50vw;
+    place-self: center;
+  }
+
+  #contact-card-title {
+    text-align: center;
+  }
+
+  .contact-form {
+    width: 50vw;
+    place-self: center;
+    margin-bottom: 100px;
+  }
+
+  #contact-btn {
+    place-self: center;
+  }
+}
+
+@media (min-width: 900px) {
```

Comment:
Too low

- [ ] Please pick a value between `992 and 1100px`  to target desktop screens 🙇‍♂️

Reply:
can you please explain a little more about image distortion of images because it's not happening on my browser?